pkg/issuer/venafi/client/request.go

```diff
@@ -182,8 +182,10 @@ func convertCustomFieldsToVcert(customFields []api.CustomField) ([]certificate.C
 	return out, nil
 }
 
-func newVRequest(cert *x509.Certificate) *certificate.Request {
+func newVRequest(cert *x509.Certificate, duration time.Duration) *certificate.Request {
 	req := certificate.NewRequest(cert)
+
+	req.ValidityDuration = &duration
```

Comment:
I think we have to also set the `IssuerHint` value when we use this field.

Reply:
There is a IssuerHintAllIssuers value that can be used to work with all CAs.

Reply:
Fixed in https://github.com/cert-manager/cert-manager/pull/7104/commits/d52fd2ff45fa67ef4ef63a4c3c1d89fcf55fbded